.github/workflows/plugin-cd.yml

```diff
@@ -41,6 +41,8 @@ jobs:
       - name: ci/build
         env:
           MM_RUDDER_PLUGINS_PROD: ${{ secrets.MM_RUDDER_PLUGINS_PROD }}
+          MM_RUDDER_CALLS_PROD: ${{ secrets.MM_RUDDER_CALLS_PROD }}
+          MM_RUDDER_DATAPLANE_URL: ${{ secrets.MM_RUDDER_DATAPLANE_URL }}
```

Comment:
If exposing these for any importing pipelines is a concern maybe we could add some logic to only set them if the repository is calls. Still doing something on the other side would be preferable but it looks more challenging.

Reply:
Let's do this now and we will figure out a proper way to do this